configuration/configuration.go

```diff
@@ -1,17 +1,22 @@
 package configuration
 
 import (
-	"github.com/cloudfoundry-incubator/multiapps-cli-plugin/ui"
 	"os"
 	"strconv"
+
+	"github.com/cloudfoundry-incubator/multiapps-cli-plugin/ui"
 )
 
 const (
-	ChunkSizeInMBEnv     = "CHUNK_SIZE_IN_MB"
-	TargetURLEnv         = "DEPLOY_SERVICE_URL"
+	// ChunkSizeInMBEnv Defines the chunk size of MTAR in MB
+	ChunkSizeInMBEnv = "CHUNK_SIZE_IN_MB"
+	// TargetURLEnv Defines the URL of the deploy service
+	TargetURLEnv = "DEPLOY_SERVICE_URL"
```

Comment:
Why is the formatter different now?

Reply:
Maybe it was not formatted before? What is the formatter for the visual studio? :D